projects/fluxcd/flux2/patches/0001-Update-go.mod-image-automation-controller-module-to-.patch

```diff
@@ -0,0 +1,97 @@
+From 8b9253e9379a2dafad4ba560162351a3a61e3b0c Mon Sep 17 00:00:00 2001
+From: Prow Bot <prow@amazonaws.com>
```

Comment:
Put your name and email here since you're the patch author, and add a signed-off field with the same.